src/read_utils.jl

```diff
@@ -2,6 +2,45 @@ using DelimitedFiles
 
 # ---------------------------------------------------------------------------------------- #
 
+struct BandSummary
+    topology :: TopologyKind
+    n :: Vector{<:Integer}
+    irlabs :: Vector{<:Integer}
+    indicator :: String
```

Comment:
```suggestion
    indicator :: Vector{Int}
```
Also: I realized you are not currently using this struct - we should do that :)